src/components/Tooltip/__tests__/Tooltip.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render, RenderResult } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { Tooltip } from '../Tooltip';
+
+describe('<Tooltip> that renders a Tooltip', () => {
+    let view: RenderResult;
+
+    it('should render default Tooltip correctly', () => {
+        view = render(
+            <Tooltip placement="bottom" content="content">
+                <>Hover me</>
+            </Tooltip>
+        );
+
+        expect(view.container).toMatchSnapshot();
+        expect(view.container).toBeInTheDocument();
```

Comment:
this check is meaning less. it is saying something like "is there a DOM in the document"... Please look for some specific element. In this case, it might be outside of the container (not the usual DOM, but it is in a Portal). To see that you could get a screenshot of `view.baseElement` (I'm not sure of the exact name).

You can also check that the tooltip actually work: you can make a user hover over it, and then check that "context" is in the document

Reply:
when i use expect(view.baseElement).toMatchSnapshot() it returns this: 
![image](https://github.com/textkernel/oneui/assets/104063210/8c7ea6cf-01e5-4dad-b85e-18aba33d1f46)